src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py

```diff
@@ -2065,7 +2065,7 @@ def _get_outbound_type(
         outbound_type = self.raw_param.get("outbound_type")
         # try to read the property value corresponding to the parameter from the `mc` object
         read_from_mc = False
-        if self.decorator_mode == DecoratorMode.CREATE:
+        if self.decorator_mode != DecoratorMode.CREATE:
```

Comment:
After the change, under update scenario, won't the value specified by user be overwritten by the existing value from mc?

Reply:
Queued test [test_aks_update_outbound_from_slb_to_natgateway](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=77089559&view=logs&j=b162b355-d59d-5864-ce0f-0a70f12dd28b&t=dc59ccd1-231f-538b-777f-33a592c7ca57&l=408), seems the update won't work

Reply:
Updated the commit and queued teshttps://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=77091936&view=results

Reply:
And it worked